diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter3/files/IndentationRule.kt

```diff
@@ -290,16 +283,10 @@ class IndentationRule(configRules: List<RulesConfig>) : DiktatRule(
     private fun ASTNode.getExceptionalIndentInitiator() = treeParent.let { parent ->
         when (parent.psi) {
             // fixme: custom logic for determining exceptional indent initiator, should be moved elsewhere
-            is KtDotQualifiedExpression -> {
-                // get the topmost expression to keep extended indent for the whole chain of dot call expressions
```

Comment:
This comment shouldn't be extracted outside of the code block. Is diktat doing this now? Seems like a bug

Reply:
May be a possible bug. I will open an issue